data-transform.qmd

```diff
@@ -625,10 +616,10 @@ The following code uses data from the **Lahman** package to compare what proport
 
 ```{r}
 batters <- Lahman::Batting |> 
-  group_by(playerID) |> 
   summarize(
     perf = sum(H, na.rm = TRUE) / sum(AB, na.rm = TRUE),
-    n = sum(AB, na.rm = TRUE)
+    n = sum(AB, na.rm = TRUE),
+    by = playerID
```

Comment:
uncaught typo of `by` vs `.by`

i dont think we can guard against this though.